ussd/core.py

```diff
@@ -449,13 +449,13 @@ def get_text(self, text_context=None):
                        else text_context
 
         if isinstance(text_context, dict):
-            language = self.ussd_request.language \
+            language = (self.ussd_request.session.get('override_language') or self.ussd_request.language) \
```

Comment:
I tried to change language. But I realized the language is saved every time a new screen is instantiated. 

That was causing the language to be changed again. 
Am not sure how well I could have done this.